nengo/utils/least_squares_solvers.py

```diff
@@ -121,7 +131,11 @@ def __call__(self, A, Y, sigma, rng=None):
 class Conjgrad(LeastSquaresSolver):
     """Solve a least-squares system using conjugate gradient."""
 
+    tol = NumberParam('tol', low=0)
+    maxiters = IntParam('maxiters', low=1, optional=True)
+
```

Comment:
I had a reason... maybe because it could be big and I didn't want to worry about hashing/pickling it? Also, it's kind of a strange parameter in general because it has to be the right size for the system being solved, meaning that when it's set the solver will only work for one size of connection. But maybe it should be included...